File: Second_Lab/task1a.cpp
#include<iostream>
#include<conio.h>
using namespace std;

int main()
{
    //declare a 2*2 matrix
    int matrix1[2][2],matrix2[2][2];
    int result[2][2];

    //input lina ko lagi
    cout<<"Enter the elements of matrix 1 :";
    for(int i=0;i<2;i++)
    {
        for(int j=0;j<2;j++)
        {
            cin>>matrix1[i][j];
        }
    }

    cout<<"Enter the elements of matrix 2 :";
    for(int i=0;i<2;i++)
    {
        for(int j=0;j<2;j++)
        {
            cin>>matrix2[i][j];
        }
    }

    //perform matrix addition
    for(int i=0;i<2;i++)
    {
        for(int j=0;j<2;j++)
        {
            result[i][j]=matrix1[i][j]+matrix2[i][j];
        }
    }

    //result
    for(int i=0;i<2;i++)
    {
        for(int j=0;j<2;j++)
        {
           cout<<result[i][j]<<" ";
        }
        cout<< endl;
    }

    return 0;
}

File: Second_Lab/task1a.py
# matrix_1 = [[1,2],[3,4]]
# matrix_2 = [[1,2],[3,4]]

# #print the matrix 
# print("Matrix 1 :")
# for row in matrix_1:
#     print(row)

# print("Matrix 2 :")
# for row in matrix_2:
#     print(row)


#user friendly 

rows=int(input("Enter the number of rows :"))
column=int(input("Enter the number of column :"))

#euta empty matrix banaunu paryo
matrix=[]


#taking input
print("Enter the elements row by row :")
for i in range(rows):
    row=list(input().split()) # if mapping use gareko bhaye harek choti usle input lai int ma convert gardinxa
    #row=list(map(int,input().split())
    matrix.append(row)

print("The entered matrix is :")
for row in matrix:
    print(row)

